data/luckyklyist.json

```diff
@@ -0,0 +1,43 @@
+{
+	"name": "Anupam Chauhan",
+	"type": "personal",
+	"bio": "Open Source DevRel |**Backend Developer** | Computer Sc Student",
+	"tags": [
+		"Open Source",
+		"Javascript",
+		"Python",
+		"Django",
+		"React"
+	],
+	"socials": [{
+			"icon": "FaTwitter",
+			"url": "https://twitter.com/luckyklyist"
+		},
+		{
+			"icon": "FaGithub",
+			"url": "https://github.com/luckyklyist"
+		},
+		{
+			"icon": "FaYoutube",
+			"url": "https://youtube.com/codewithklyist"
```

Comment:
```suggestion
			"url": "https://youtube.com/@codewithklyist"
```

Reply:
nice catch, @imbeshat

Reply:
Hey @luckyklyist, do you need any help in making the above requested changes!?